pkg/apis/serving/v1beta1/predictor_tfserving.go

```diff
@@ -88,6 +88,7 @@ func (t *TFServingSpec) GetContainer(metadata metav1.ObjectMeta, extensions *Com
 		fmt.Sprintf("%s=%s", "--rest_api_port", TensorflowServingRestPort),
 		fmt.Sprintf("%s=%s", "--model_name", metadata.Name),
 		fmt.Sprintf("%s=%s", "--model_base_path", constants.DefaultModelLocalMountPath),
+		fmt.Sprintf("%s=%s", "--rest_api_timeout_in_ms", config.Predictors.Tensorflow.DefaultTimeout),
```

Comment:
@theofpa [timeout](https://github.com/kubeflow/kfserving/blob/master/pkg/apis/serving/v1beta1/component.go#L80) is already added in v1beta1 API, the value needs to match user specified value, the defaultTimeout should be applied on mutating webhook.

Reply:
addressed in 565d3ab